metomi/rose/metadata_graph.py

```diff
@@ -294,9 +294,46 @@ def main():
     """Run the metadata graphing from the command line."""
     _load_override_config()
     metomi.rose.macro.add_meta_paths()
-    opt_parser = metomi.rose.opt_parse.RoseOptionParser()
-    options = ["conf_dir", "meta_path", "output_dir", "property"]
-    opt_parser.add_my_options(*options)
+    opt_parser = metomi.rose.opt_parse.RoseOptionParser(
+        usage='rose metadata-graph [OPTIONS] [SECTION ...]',
```

Comment:
I can't preview this cli docs as I get 
```
[~/rose]$ rose metadata-graph --help
This command has been removed pending re-implementation
```
I'm assuming this is ready in place for reimplementation.